### PR TITLE
Added Github section in Devops resources

### LINKED ISSUE #425: 💡 [FEATURE] -  Add GitHub Repository Section in DevOps section
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

In DevOps Resources , there is no GitHub repository link , where someone will explore how it works , I want to add this feature.

### Explain your solution

I want to add a section called GitHub repository where i will share the link of many GitHub repo where someone will find many projects of DevOps , for beginners to advance level.

### Any alternative approaches/features

_No response_

### Additional Context
I want to contribute in this issue , will you assign this to me @jfmartinz .

### PR DESCRIPTION
**Pull request by Pujan-sarkar**
- `DevOps- `

Fixes #425 

### Description: 
I have added a section named GitHub repo in DevOps resources section , where i divided them into three categories Beginner, intermediate, Advance.

### Checklist:

- [X] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [X] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Review by jfmartinz — CHANGES_REQUESTED** [review]
[Awesome Scalability](https://github.com/awesome-scalability/awesome-scalability) Is not working, 404. Also remove the beginner, intermediate, and advanced thanks

**Comment by Pujan-sarkar** [comment]
@jfmartinz  i have done the changes , please check it and give me the feedback.

File: DevOps/readme.md [hunk 1]
```diff
@@ -316,3 +317,57 @@ DevOps is a combination of software development (Dev) and operations (Ops). It i
...
+### GitHub Repositories
+
+<table width="100%">
+    <tr>
+        <th>Resource</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td colspan="2"><b>Beginner</b></td>
+    </tr>
```
Comment (jfmartinz) [inline DevOps/readme.md:330]:
```suggestion

```

File: DevOps/readme.md [hunk 2]
```diff
@@ -316,3 +317,57 @@ DevOps is a combination of software development (Dev) and operations (Ops). It i
...
+        <td><a href="https://github.com/opsclass/ops-class.org">ops-class.org</a></td>
+        <td>Offers educational resources and materials on operating systems, which is fundamental for beginners starting in DevOps.</td>
+    </tr>
+    <tr>
+        <td><a href="https://github.com/kamranahmedse/developer-roadmap">Developer Roadmap</a></td>
+        <td>Provides a roadmap for developers, including paths for becoming a DevOps engineer. It is a great resource for beginners to understand the skills and tools needed in DevOps.</td>
+    </tr>
+    <tr>
+        <td colspan="2"><b>Intermediate</b></td>
+    </tr>
```
Comment (jfmartinz) [inline DevOps/readme.md:345]:
```suggestion

```

File: DevOps/readme.md [hunk 3]
```diff
@@ -316,3 +317,57 @@ DevOps is a combination of software development (Dev) and operations (Ops). It i
...
+        <td><a href="https://github.com/earthly/awesome-earthly">Awesome Earthly</a></td>
+        <td>A curated list of resources about Earthly, a build automation tool that simplifies CI/CD pipelines. Suitable for intermediate DevOps engineers looking to streamline their build processes.</td>
+    </tr>
+    <tr>
+        <td><a href="https://github.com/open-guides/og-aws">Open Guide to AWS</a></td>
+        <td>A comprehensive guide to using Amazon Web Services (AWS), covering best practices, tips, and tools. Ideal for intermediate DevOps professionals working with AWS.</td>
+    </tr>
+    <tr>
+        <td colspan="2"><b>Advanced</b></td>
+    </tr>
```
Comment (jfmartinz) [inline DevOps/readme.md:360]:
```suggestion
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
404 found:

https://github.com/awesome-scalability/awesome-scalability
https://github.com/opsclass/ops-class.org

You can either remove or change the URL

**Comment by Pujan-sarkar** [comment]
@jfmartinz okay... Can i close this PR and send another with the changes you want  ?

**Comment by jfmartinz** [comment]
No need.

**Comment by jfmartinz** [comment]
Just remove those cols, beginner, intermediate, and advanced. And remove those URL that 404 or change them

**Pull request closed without merging**